### PR TITLE
feat: add building unit HasDeviation to projections

### PR DESCRIPTION
**Pull request by jvandaal**

### CODE REVIEW

**Review by emalfroy — DISMISSED** [review]

File: src/BuildingRegistry.Projections.Extract/BuildingUnitExtract/BuildingUnitDbaseRecord.cs [hunk 1]
```diff
@@ -13,6 +13,7 @@ public class BuildingUnitDbaseRecord : DbaseRecord
         public DbaseCharacter functie { get; }
         public DbaseCharacter status { get; }
         public DbaseCharacter posgeommet { get; }
+        public DbaseBoolean afwijkingvastgesteld { get; }
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Extract/BuildingUnitExtract/BuildingUnitDbaseRecord.cs:16]: Just `afwijking`

File: src/BuildingRegistry.Projections.Extract/BuildingUnitExtract/BuildingUnitDbaseSchema.cs [hunk 2]
```diff
@@ -11,6 +11,7 @@ public class BuildingUnitDbaseSchema : DbaseSchema
         public DbaseField functie => Fields[4];
         public DbaseField status => Fields[5];
         public DbaseField posgeommet => Fields[6];
+        public DbaseField afwijkingvastgesteld => Fields[7];
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Extract/BuildingUnitExtract/BuildingUnitDbaseSchema.cs:14]: `afwijking`

File: src/BuildingRegistry.Projections.Extract/Migrations/20221115074040_AddBuildingUnitHasDeviation.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BuildingRegistry.Projections.Extract.Migrations
+{
+    public partial class AddBuildingUnitHasDeviation : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Extract/Migrations/20221115074040_AddBuildingUnitHasDeviation.cs:9]: no migration needed

**Review by ArneD — CHANGES_REQUESTED** [review]

**Review by ArneD — APPROVED** [review]

**Pull request merged**